desktop/app.js

```diff
@@ -46,11 +47,6 @@ module.exports = function main() {
       defaultHeight: 768,
     });
 
-    // Create the browser window.
-    const iconPath = path.join(
-      __dirname,
-      '../lib/icons/app-icon/icon_256x256.png'
```

Comment:
Does this mean this icon is now unused, or is it being loaded somewhere else? I would like to remove it if possible (see #2158)